Cargo.toml

```diff
@@ -66,6 +66,10 @@ snap = "1.0.3"
 tokio = { version = "1.8.2", features = ["rt-multi-thread", "signal", "test-util", "parking_lot"] }
 tokio-stream = "0.1.2"
 tonic = "0.5.0"
+tracing = {version = "0.1"}
+tracing-subscriber = {version = "0.2"}
+tracing-futures = { version = "0.2.5", default-features = false }
+tracing-test = "0.1"
```

Comment:
Since this is a test dependency it should be added under `dev-dependencies` instead